app/enterprise/1.3-x/deployment/installation/amazon-linux-2.md

```diff
@@ -0,0 +1,338 @@
+
+
+
+## Step 1. Download Kong Enterprise
+
+1. Download the RPM package from Bintray
+  ```
+  $ sudo yum update
+  $ wget 'https://<BINTRAY_USERNAME:<PASSWORD>@bintray.com/kong/kong-enterprise-edition-aws/rpm' -O bintray-kong-kong-enterprise-edition-aws.repo --auth-no-challenge`
+  ```
+  > Note: this command requires your Bintray account PASSWORD, **not** your API KEY, 
+  > If you receive the error “BAD PORT” ensure your password does not contain any shell meta-characters like $,@, or / 
+
+2. Move the package into `/etc/yum.repos.d/`
+  ```
+  $ sudo mv bintray-kong-kong-enterprise-edition-aws.repo /etc/yum.repos.d/
+  ```
+
+3. Open the package to and add your Bintray credentials to the `baseurl` value (See [Getting your Bintray Credentials] for help)
+  ```
+  $ sudo vi /etc/yum.repos.d/bintray-kong-kong-enterprise-edition-aws.repo
+  ```
+  ```
+  baseurl=https://<BINTRAY_USER>:<BINTRAY_API_KEY>@kong.bintray.com/kong-enterprise-edition-aws
+  ```
+
+4. Obtain your Kong Enterprise license file
+
+  If you do not already have your license file, you can download it from your account files in Bintray *`https://bintray.com/kong/<YOUR_REPO_NAME>/license#files`. *See [How you Access Your Enterprise License](/enterprise/latest/deployment/access-license) for help.
+
+
+    Ensure your license file is in proper *`JSON`*:
+
+    ```
+    {"license":{"signature":"91e6dd9716d12ffsn4a5ckkb16a556dbebdbc4d0a66d9b2c53f8c8d717eb93dd2bdbe2cb3ef51c20806f14345128907da35","payload":{"customer":"Kong Inc","license_creation_date":"2019-05-07","product_subscription":"Kong Enterprise Edition","admin_seats":"5","support_plan":"None","license_expiration_date":"2021-04-01","license_key":"00Q1K00000zuUAwUAM_a1V1K000005kRhuUAE"},"version":1}}
+    ```
+
+5. Securely copy the license file to the EC2 instance
+  ```
+  $ scp license.json <ec2-username>@<serverip>:~
+  ```
+
+
+## Step 2. Install Kong Enterprise
+
+1. Install Kong Enterprise
+  ```
+  $ sudo yum install kong-enterprise-edition
+  ```
+
+2. Copy the license file to `/etc/kong/license.json`
+  ```
+  $ sudo cp kong-se-license.json /etc/kong/license.json
+  ```
+  Kong will look for a valid license in this location.
+
+
+## Step 3. Install and Initialize the Database
+
+**Kong Enterprise** on Amazon Linux 2 supports:
+
+- [PostgreSQL](/#postgresql)
+- Cassandra
+
+
+### PostgreSQL
+
+This configuration supports PostgreSQL 9.6, 10, and 11 via [Amazon Linux Extras](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/amazon-linux-ami-basics.html#extras-library).
+
+
+**Steps for this section:**
+
+- i. [Install and Start Postgres](/#install-and-start-postgresql)
+- ii. [Create a Kong user and database](/#create-a-kong-user-and-database)
+- iii. [Connect Kong to the Postgres database](/#connect-kong-to-the-postgresql-database)
+
+
+**i. Install and Start PostgreSQL**
+
+1. Install PostgreSQL
+  ```
+  $ sudo amazon-linux-extras install postgresql9.6
+  or
```

Comment:
I can't easily make the suggestion since the code block would break, but I'd move "or" outside of the code block.

```
 $ sudo amazon-linux-extras install postgresql9.6
```

or

```
$ sudo amazon-linux-extras install posgresql10
```

or 

```
$ sudo amazon-linux-extras install postgresql11
```

Reply:
I applied this change, up for suggestions on making this section more userfriendly as the commands for 9.6 and 10 are slightly different :/ once we move to a system that allows for code snippets this section should be expanded

Reply:
K, it's good for now, that was merely a pref. Thanks! :)